scripts/local/launch/start_dotnet_only_services.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
We are using `-` in the file names as separator rather than `_`. Can you please change it for consistency?